src/components/Map.vue

```diff
@@ -49,6 +49,8 @@ export default {
   props: {
     detectives: Array,
     currentPlayer: Object || null,
+    isMrXVisible: Boolean,
+    mrXLocation: Number || Boolean,
```

Comment:
it can be false, when Mr.X is not visible, so we need to account for the extra type

Reply:
instead of tagging the boolean to mrXLocation, can isMrXVisible itself be used?

Reply:
unfortunately that would involve passing the location of Mr. X to the Map at all times, which I didn't want to do (not that there's any issue with it)

this seemed cleaner to me, since Mr. X location is either known (Number) or unknown (false Boolean)

but I can be convinced otherwise.